### PR TITLE
R pidkostelnii add news frontend/businessLogic

### PR DESCRIPTION
**Pull request by roman454647**

### CODE REVIEW

File: client/src/containers/AddNews/component/style.scss [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+      &:hover {
+        cursor: url('../../../assets/img/cursor-hover.svg'), auto;
+      }
+    }
+  }
+
+
+  .alt-img-name,
+  .title,
```
Comment (anna-kuzii) [inline client/src/containers/AddNews/component/style.scss]: Bad names

File: client/src/containers/AddNews/component/style.scss [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+    width: 100%;
+    height: 400px;
+    margin: 20px 0;
+    background-color: $grey-08;
+
+    .red-circle {
```
Comment (anna-kuzii) [inline client/src/containers/AddNews/component/style.scss]: Today — red, tomorrow — blue

File: client/src/containers/AddNews/component/style.scss [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+      font-family: Open Sans;
+      font-size: 12px;
+      font-style: normal;
+      font-weight: 600;
+      line-height: 16px;
+      color: $ghost;
+      border: 1px solid $mischka;
+      box-sizing: border-box;
+
+      &[placeholder] {
```
Comment (anna-kuzii) [inline client/src/containers/AddNews/component/style.scss]: What about pseudo class?

File: client/src/containers/AddNews/reducer.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    case articleConstants.SET_NEW_ARTICLE_REQUEST:
+      return {
+        adding: true,
+        ...state,
+      };
+    case articleConstants.SET_NEW_ARTICLE_SUCCESS:
+      return {
+        ...state,
+        adding: false,
+        error: null,
```
Comment (anna-kuzii) [inline client/src/containers/AddNews/reducer.js]: We don’t need error here

File: client/src/styles/variables.scss [hunk 5]
```diff
@@ -20,7 +20,6 @@ $silver-chalice: #b2b2b2;
 $ebony-clay: #2b2f43;
 $concrete: #f2f2f2;
 $mine-shaft: #2f2f2f;
-$flush-mahogany: #c63638;
```
Comment (anna-kuzii) [inline client/src/styles/variables.scss]: Are u sure?

Comment (roman454647) [reply client/src/styles/variables.scss]: no,changed

Comment (roman454647) [reply client/src/styles/variables.scss]: yes, it was duplicated

File: client/src/containers/AddNews/reducer.js (hunk 4 above)
Comment (roman454647) [reply client/src/containers/AddNews/reducer.js]: deleted

File: client/src/containers/AddNews/component/style.scss (hunk 2 above)
Comment (roman454647) [reply client/src/containers/AddNews/component/style.scss]: changed to circle

File: client/src/containers/AddNews/component/style.scss (hunk 3 above)
Comment (roman454647) [reply client/src/containers/AddNews/component/style.scss]: used

File: client/src/styles/variables.scss (hunk 5 above)
Comment (roman454647) [reply client/src/styles/variables.scss]: yes, it was duplicated

File: client/src/containers/AddNews/component/style.scss (hunk 1 above)
Comment (roman454647) [reply client/src/containers/AddNews/component/style.scss]: changed

**Pull request merged**